### DISCUSSION TITLE
Debug Certificate Issues

### DISCUSSION THREAD

**Discussion by cwages** [codefori/vscode-ibmi#2030] [General]
After updating to code for ibm i v2.10.0, All of our developers started getting this message.
![image](https://github.com/codefori/vscode-ibmi/assets/168861539/7be4fc81-8474-4401-be4f-2f25517edac2)

These certificates were initially generated at a previous version of the extension by our operations team. Which has all object authority.
Our regular developer profiles do not.

Based on this, what should we do to get this usable for all of our developers again?

**Comment by sebjulliand** [comment]
Hi @cwages ,
please have someone from your operations team to connect with Code for IBM i v2.10.
The certificates previously generated will be migrated to the new format and this warning will disappear.

Let us know if you still have issues after that.

**Comment by cwages** [comment]
That got it going, thanks for your quick response.

**Reply by sebjulliand** [reply]
Excellent! You're welcome!
Now the certificates and the Debug Service configuration have been migrated, you can now use `SBMJOB` to start the service: https://codefori.github.io/docs/developing/debug/#starting-the-debug-service-outside-of-code-for-ibm-i
That can be useful to restart the service after an IPL, for example 😉
